test/inductor/test_torchinductor.py

```diff
@@ -2693,6 +2693,34 @@ def fn(x):
             (torch.randn([64]),),
         )
 
+    def test_expm1(self):
+        def fn(x):
+            return torch.expm1(x), torch.expm1(x) * 2
+
+        for dtype in (torch.float16, torch.float, torch.double, torch.int, torch.int64):
+            self.common(
+                fn,
+                (torch.randn([64]).to(dtype=dtype),),
+            )
+            self.common(
+                fn,
+                (torch.arange(-0.00001, 0.00001, 0.001).to(dtype=dtype),),
```

Comment:
the step here is too large, you end up with a single value, -1e-5

Reply:
oops, good catch